pkg/volume/csimigration/plugin_manager.go

```diff
@@ -60,18 +60,6 @@ func (pm PluginManager) IsMigrationCompleteForPlugin(pluginName string) bool {
 	}
 
 	switch pluginName {
-	case csilibplugins.AWSEBSInTreePluginName:
-		return pm.featureGate.Enabled(features.InTreePluginAWSUnregister)
-	case csilibplugins.GCEPDInTreePluginName:
-		return pm.featureGate.Enabled(features.InTreePluginGCEUnregister)
-	case csilibplugins.AzureFileInTreePluginName:
-		return pm.featureGate.Enabled(features.InTreePluginAzureFileUnregister)
-	case csilibplugins.AzureDiskInTreePluginName:
-		return pm.featureGate.Enabled(features.InTreePluginAzureDiskUnregister)
-	case csilibplugins.CinderInTreePluginName:
-		return pm.featureGate.Enabled(features.InTreePluginOpenStackUnregister)
-	case csilibplugins.VSphereInTreePluginName:
-		return pm.featureGate.Enabled(features.InTreePluginvSphereUnregister)
```

Comment:
The function should return `true` for these cases.

Reply:
There's another PR to promote these feature gates to GA and lock it by default. https://github.com/kubernetes/kubernetes/pull/124023.  https://github.com/kubernetes/kubernetes/pull/124108 is to fix this test failure in https://github.com/kubernetes/kubernetes/pull/124023. 

Should we promote or delete this feature-gate from the codebase? @jsafrane

Reply:
If the promotion is desired, this PR can be closed.

Reply:
I added it to today's meeting agenda. I would personally delete them.

Reply:
We decided to continue with this PR and delete the feature gates right away. Please fix this comment and we merge it tomorrow.